stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -1349,10 +1349,10 @@ public func _setDownCast<BaseValue, DerivedValue>(_ source: Set<BaseValue>)
   && _isClassOrObjCExistential(DerivedValue.self) {
     switch source._variantStorage {
     case _VariantSetStorage.native(let storage):
-      let bridgingStorage = storage.bridgingStorage()
+      let bridged = storage.bridged()
       return Set(
         _immutableCocoaSet:
-        unsafeBitCast(bridgingStorage, to: _NSSet.self))
+        unsafeBitCast(bridged, to: _NSSet.self))
```

Comment:
Can you elaborate on what the correct code for this might be? I'm still learning all the different cast idioms here. Gut: changed .bridged() to return _NSSet.